.github/workflows/dev_tests.yml

```diff
@@ -0,0 +1,46 @@
+name: dev tests
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
suggest adding a workflow_dispatch option here for easier on-demand testing if a test fails and we need to make sure it actually fails.